docs/design/README.md

```diff
@@ -17,24 +17,24 @@ It's designed to run in the master plane of a Kubernetes cluster. It follows the
 ## Objects of Node Controller Manager
 
 Node Controller Manager makes use of 4 CRD objects and 1 Kubernetes secret object to manage machines. They are as follows,
-1. Instance-class: A template that contains cloud provider specific details used to create instances.
-1. Instance: Node Controller Manager's object that actually represents a VM.
-1. Instance-set: A group of instances managed by the Node Controller Manager. 
-1. Instance-deploy: A group of instance-sets managed by the Node Controller Manager to allow updating instances.
+1. Machine-class: A template that contains cloud provider specific details used to create machines.
+1. Machine: Node Controller Manager's object that actually represents a VM.
+1. Machine-set: A group of machines managed by the Node Controller Manager. 
+1. Machine-deploy: A group of machine-sets managed by the Node Controller Manager to allow updating machines.
```

Comment:
Machine-set: Responsible for maintaining number of Healthy machines.
Machine-deployment: Responsible for managing Machine-sets to achieve Roliilngupdate/Recreate of machine versions.